invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/CommunityProfileForm.js

```diff
@@ -587,7 +594,11 @@ class CommunityProfileForm extends Component {
               </Grid.Row>
               <Grid.Row className="danger-zone">
                 <Grid.Column as="section" width={16}>
-                  <DangerZone community={community} onError={this.setGlobalError} />
+                  <DangerZone
```

Comment:
Missing hiding the entire danger zone if no permissions. Can you please wrap (and test) the `<Grid.Row` above with:
```javascript
      {(permissions.can_rename || permissions.can_delete) && (
```

Reply:
I added this to the `DangerZone` cmp instead, in case it's reused